### DISCUSSION TITLE
v1.5.0 Upgrade discussion

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#5439] [Announcement]
If you have any question wrt the [v1.5.0 release and upgrade](https://github.com/filecoin-project/lotus/discussions/5617), leave a comment here!

**Comment by gopherliu** [comment]
after 2021-02-27, strongly recommend

**Reply by jennijuju** [reply]
Strong recommendation without any reason doesnt make the recommendation so strong. I assume this is suggested due to the CNY which it will be Chinese holiday from Feb 11-17 and we definitely have this in mind. The current proposed plan is the release will be out on Feb 1st, and the upgrade epoch is ~ a week later.

**Reply by gopherliu** [reply]
Exactly ~ The Spring Festival is a traditional Chinese festival.

**Comment by liujianghu** [comment]
after 2021-02-28, strongly recommend

**Comment by sky1737** [comment]
after 2021-03-01, strongly recommend

**Comment by RyanZhangGodLike** [comment]
after 2021-03-01, strongly recommend

**Comment by fmtmbs** [comment]
I suggest to upgrade it before Feb 5th or after Feb 20th, just let us have a happy holiday.

**Comment by deaswang** [comment]
Upgrade v1.5.0 on Calibration network before Festival. upgrade Mainnet after Festival?

**Comment by yuanxinw-network** [comment]
I suggest to upgrade after 28th Feb, we need a happy new year. 
Compared with the new features, we We need a stable network even more

**Comment by yuyuhe123** [comment]
I suggest that the update be made after March 1st.

**Comment by johnchenyan** [comment]
after 2021-03-01, strongly recommend

**Comment by XQYCHJ** [comment]
Year, we just want a happy relaxing spring festival, updating after 02-28 is a easy-to-accept solution. Strongly recommend.

**Comment by gnasnik** [comment]
after 2021-03-01, strongly recommend

**Comment by beck-8** [comment]
after 2021-03-01, strongly recommend

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
june. april is easter.

**Reply by jennijuju** [reply]
🤡

**Comment by flyworker** [comment]
Early Feb is fine for us

**Comment by NSC-FIL** [comment]
Anytime is fine as long as all attack vectors have been tested and v1.5.0 is deemed stable in calibration-net before releasing it.

**Comment by deltazxm** [comment]
Feb 1st is the best.We have enough time to deal with before the Spring Festival.

**Comment by jianghan1976** [comment]
after 2021-03-01, strongly recommend

**Comment by yungo-sz** [comment]
Could you please tell me how to verify the Miner's SubmitWindowedPost message that I would like to earn some rewards
